src/System.Data.OleDb/src/Resources/Strings.resx

```diff
@@ -0,0 +1,1020 @@
+  <data name="DataCategory_NamedConnectionString" xml:space="preserve">
+    <value>Named ConnectionString</value>
+  </data>
+  <data name="DataCategory_Security" xml:space="preserve">
+    <value>Security</value>
+  </data>
+  <data name="DataCategory_Source" xml:space="preserve">
+    <value>Source</value>
+  </data>
+  <data name="SqlConvert_ConvertFailed" xml:space="preserve">
+    <value>Cannot convert object of type '{0}' to object of type '{1}'.</value>
+  </data>
+  <data name="ADP_InvalidConnectionOptionValue" xml:space="preserve">
+    <value>Invalid value for key '{0}'.</value>
+  </data>
+  <data name="ADP_KeywordNotSupported" xml:space="preserve">
+    <value>Keyword not supported: '{0}'.</value>
+  </data>
+  <data name="ADP_InternalProviderError" xml:space="preserve">
+    <value>Internal .Net Framework Data Provider error {0}.</value>
+  </data>
+  <data name="ADP_InvalidMultipartName" xml:space="preserve">
+    <value>{0} "{1}".</value>
+  </data>
+  <data name="ADP_InvalidMultipartNameQuoteUsage" xml:space="preserve">
+    <value>{0} "{1}", incorrect usage of quotes.</value>
+  </data>
+  <data name="ADP_InvalidMultipartNameToManyParts" xml:space="preserve">
+    <value>{0} "{1}", the current limit of "{2}" is insufficient.</value>
+  </data>
+  <data name="OLEDB_OLEDBCommandText" xml:space="preserve">
+    <value>OleDbCommandBuilder.DeriveParameters failed because the OleDbCommandBuilder.CommandText property value is an invalid multipart name</value>
+  </data>
+  <data name="ADP_NotSupportedEnumerationValue" xml:space="preserve">
+    <value>The {0} enumeration value, {1}, is not supported by the {2} method.</value>
+  </data>
+  <data name="ADP_InvalidSourceBufferIndex" xml:space="preserve">
+    <value>Invalid source buffer (size of {0}) offset: {1}</value>
+  </data>
+  <data name="ADP_InvalidDestinationBufferIndex" xml:space="preserve">
+    <value>Invalid destination buffer (size of {0}) offset: {1}</value>
+  </data>
+  <data name="ADP_StreamClosed" xml:space="preserve">
+    <value>Invalid attempt to {0} when stream is closed.</value>
+  </data>
+  <data name="ADP_InvalidSeekOrigin" xml:space="preserve">
+    <value>Specified SeekOrigin value is invalid.</value>
+  </data>
+  <data name="OleDb_SchemaRowsetsNotSupported" xml:space="preserve">
+    <value>'{0}' interface is not supported by the '{1}' provider.  GetOleDbSchemaTable is unavailable with the current provider.</value>
+  </data>
+  <data name="OleDb_NoErrorInformation2" xml:space="preserve">
+    <value>'{0}' failed with no error message available, result code: {1}.</value>
+  </data>
+  <data name="OleDb_NoErrorInformation" xml:space="preserve">
+    <value>No error message available, result code: {0}.</value>
+  </data>
+  <data name="OleDb_MDACNotAvailable" xml:space="preserve">
+    <value>The .Net Framework Data Providers require Microsoft Data Access Components(MDAC).  Please install Microsoft Data Access Components(MDAC) version 2.6 or later.</value>
```

Comment:
If this string (and the two below that mention .NET Framework) are not actually dead, please remove mention of .NET Framework and use some general term.